### PR TITLE
Added step for auto-download and unzip of VST3 SDK to CMakeLists.txt.

### PR DESCRIPTION
**Pull request by LeStahL**
This will check at configure time if the VST3 SDK is present, and if not: download & unpack it into the correct folder.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,7 +8,19 @@ if(MSVC)
...
+
+# Download and unpack VST3 SDK
+find_file(VST3SDK_TEST name public.sdk HINTS ${VSTSDK3_DIR})
+if(${VST3SDK_TEST} MATCHES VST3SDK_TEST-NOTFOUND)
+	message(STATUS "VST3 SDK not found. Will download.")
+	file(DOWNLOAD https://web.archive.org/web/20200502121517/https://www.steinberg.net/sdk_downloads/vstsdk366_27_06_2016_build_61.zip vstsdk366_27_06_2016_build_61.zip SHOW_PROGRESS)
+	file(ARCHIVE_EXTRACT INPUT vstsdk366_27_06_2016_build_61.zip DESTINATION vstsdk366_27_06_2016_build_61 VERBOSE)
+	file(GLOB VST3SDK_FILES "${CMAKE_CURRENT_BINARY_DIR}/vstsdk366_27_06_2016_build_61/VST3\ SDK/*")
+	file(COPY ${VST3SDK_FILES} DESTINATION ${VSTSDK3_DIR})
+endif()
```
Comment (kusma) [inline CMakeLists.txt:23]: I'm skeptical of build systems that downloads and compiles code behind my back, without giving me a reasonable chance to intervene. IMO, auto-downloading should be an opt-in feature.

Comment (LeStahL) [reply CMakeLists.txt:24]: Adjusted accordingly.

File: CMakeLists.txt [hunk 2]
```diff
@@ -10,6 +10,19 @@ endif()
 
 set(VSTSDK3_DIR "${CMAKE_SOURCE_DIR}/Vst3.x/" CACHE PATH "VSTSDK location")
 
+# Download and unpack VST3 SDK
+set(DOWNLOAD_DEPENDENCIES CACHE BOOL OFF)
```
Comment (yupferris) [inline CMakeLists.txt:14]: I'd like this to be called `DOWNLOAD_VST3SDK`.

**Review by yupferris — CHANGES_REQUESTED** [review]
Cheers for this! Only one minor change, and I'll be happy to accept it :)

File: CMakeLists.txt (hunk 2 above)
Comment (LeStahL) [reply CMakeLists.txt]: Done.

**Review by yupferris — APPROVED** [review]
cheers!

**Pull request merged**